package.json

```diff
@@ -92,8 +92,7 @@
     "netlify:build:with_storybook": "yarn netlify:build && yarn storybook:build && mv storybook-static public/storybook",
     "storybook": "start-storybook -p 6006",
     "storybook:build": "build-storybook --quiet",
-    "test:update": "yarn test --updateSnapshot",
-    "test": "jest --json --outputFile=.jest-test-results.json"
+    "test": "jest --json --updateSnapshot --outputFile=.jest-test-results.json"
```

Comment:
This feels unrelated to this work. It also doesn't seem like a good direction. The point of having the snapshots is to make sure that nothing changed. If we update them every time that defeats the purpose, doesn't it?